src/alias.rs

```diff
@@ -14,14 +14,18 @@ pub fn create_alias(
 
     let version_dir = version
         .installation_path(config)
-        .ok_or_else(|| std::io::Error::from(std::io::ErrorKind::NotFound))?;
+        .ok_or_else(|| std::io::Error::from(std::io::ErrorKind::NotFound));
```

Comment:
Can we move this block to be at line 25, so declaration and usage will be very close?